src/cloudflare/internal/streaming-base64.ts

```diff
@@ -0,0 +1,194 @@
+import base64 from 'cloudflare-internal:base64';
+
+function base64Error(cause: unknown): Error {
+  // @ts-expect-error: Error.isError seems to be missing from types?
```

Comment:
This is going to be enabled after 5.9 version of Typescript, which is not yet released. Can you add a TODO(soon) here as well, so I can update this once we update TS?